### PR TITLE
gnupg 2.2.22

### PR DESCRIPTION
**Pull request by miccal**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

File: Formula/gnupg.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Gnupg < Formula
   desc "GNU Pretty Good Privacy (PGP) package"
   homepage "https://gnupg.org/"
-  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.21.tar.bz2"
-  sha256 "61e83278fb5fa7336658a8b73ab26f379d41275bb1c7c6e694dd9f9a6e8e76ec"
-  license "GPL-3.0"
+  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.22.tar.bz2"
+  sha256 "7c1370565e1910b9d8c4e0fb57b9de34aa062ec7bb91abad5803d791f38d855b"
+  license "GPL-3.0-only"
```
Comment (chenrui333) [inline Formula/gnupg.rb:6]: only reference for this change, I can see it actually use `GPL-3.0-or-later`? cc @Rylan12

**Comment by chenrui333** [comment]
here is the license header ref I find:

```
/* findkey.c - Locate the secret key
 * Copyright (C) 2001, 2002, 2003, 2004, 2005, 2007,
 *               2010, 2011 Free Software Foundation, Inc.
 * Copyright (C) 2014 Werner Koch
 *
 * This file is part of GnuPG.
 *
 * GnuPG is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 3 of the License, or
 * (at your option) any later version.
 *
 * GnuPG is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, see <https://www.gnu.org/licenses/>.
 */
```

**Comment by chenrui333** [comment]
here are some files (`argparse.c` and `agent-opt.c`) are dual-licensed:

```
/* [argparse.c wk 17.06.97] Argument Parser for option handling
 * Copyright (C) 1998-2001, 2006-2008, 2012 Free Software Foundation, Inc.
 * Copyright (C) 1997-2001, 2006-2008, 2013-2017 Werner Koch
 *
 * This file is part of GnuPG.
 *
 * GnuPG is free software; you can redistribute and/or modify this
 * part of GnuPG under the terms of either
 *
 *   - the GNU Lesser General Public License as published by the Free
 *     Software Foundation; either version 3 of the License, or (at
 *     your option) any later version.
 *
 * or
 *
 *   - the GNU General Public License as published by the Free
 *     Software Foundation; either version 2 of the License, or (at
 *     your option) any later version.
 *
 * or both in parallel, as here.
 *
 * GnuPG is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * General Public License for more details.
 *
 * You should have received a copies of the GNU General Public License
 * and the GNU Lesser General Public License along with this program;
 * if not, see <https://gnu.org/licenses/>.
 */

/* This file may be used as part of GnuPG or standalone.  A GnuPG
   build is detected by the presence of the macro GNUPG_MAJOR_VERSION.
   Some feature are only availalbe in the GnuPG build mode.
 */

```

**Comment by chenrui333** [comment]
b64enc file is with `LGPL-2.1-or-later`

File: Formula/gnupg.rb (hunk 1 above)
Comment (miccal) [reply Formula/gnupg.rb:6]: Sorry @chenrui333, fixing now.

Comment (chenrui333) [reply Formula/gnupg.rb]: @miccal no need to apologize. appreciate your work in here. ❤️

Comment (chenrui333) [reply Formula/gnupg.rb]: It is a wild land for applying the license, I might still need extra eyes on the license change. 🙈

**Review by chenrui333 — DISMISSED** [review]
let's move forward with this, we might need a separate license PR to clear it out.

**Comment by SMillerDev** [comment]
@miccal can you remove the license field for now? So we can add the proper one later.

**Comment by miccal** [comment]
@SMillerDev done, thank you.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**